src/main/java/com/sonyericsson/jenkins/plugins/bfa/tokens/Renderer.java

```diff
@@ -297,9 +318,17 @@ private void addIndicationRepresentation(final StringBuilder stringBuilder,
             stringBuilder.append(":\n");
             stringBuilder.append(indentForDepth(indentLevel));
             stringBuilder.append(LIST_BULLET_SPACE);
-            stringBuilder.append("<");
+            if (escapeHtml) {
+                stringBuilder.append("&lt;");

```

Comment:
If the plugin is adding this itself, is it always safe? Or can we not guarantee that? I'm confused about why we would emit `&lt;` here vs just not emitting anything.

Reply:
I am interpreting the situation as the user wants the text version of the output, but still want it to be safe html :man_shrugging: